test/unit/dir.html

```diff
@@ -109,6 +109,28 @@
     </script>
   </dom-module>
 
+  <dom-module id="x-complicated">
+    <template>
+      <style>
+        #target {
+          padding: 20px;
+          color: white;
+          background: rgb(0, 0, 255);
+        }
+        :dir(rtl) #target {
```

Comment:
Add test for `#target:dir(rtl)`, since this is actually the correct selector for what people will typically want.

Need to make sure users are writing `:dir()` as a compound selector, not a descendant selector.  cc @arthurevans @sorvell @bicknellr @valdrinkoshi

Reply:
FYI did some tests with the compound selector, it works fine with most cases except when applied to another custom element, e.g. `my-element:dir(rtl)`: http://jsbin.com/cewijoy/1/edit?html,output

FWIW, testing descendant selectors on FF (where `:dir` is implemented natively) works as advertised.

Reply:
Filed the compound selector issue here https://github.com/Polymer/polymer/issues/4966

Reply:
@kevinpschaaf Test added

@valdrinkoshi Can you file a bug with that example separately?

Reply:
done already, #4966